navapbc/labs-decision-support-tool
# bug: fix accordion

## Ticket

N/A

## Changes
Type cast floats to str


## Context for reviewers
Before type change
<img width="866" alt="Screenshot 2024-08-12 at 4 24 39 PM" src="https://github.com/user-attachments/assets/1d2f158c-3e77-4e5e-8565-5b529dda39d2">
After type change
<img width="772" alt="Screenshot 2024-08-12 at 4 25 46 PM" src="https://github.com/user-attachments/assets/05e0c7df-c3d9-4edd-bec2-3b35eee37581">


## Testing
- navigate to http://localhost:8000/chat/?engine=bridges-eligibility-manual
- enter query, text should be within accordion
